packages/@react-spectrum/listbox/src/ListBoxBase.tsx

```diff
@@ -67,6 +72,10 @@ function ListBoxBase<T>(props: ListBoxBaseProps<T>, ref: RefObject<HTMLDivElemen
     isVirtualized: true
   }, state);
   let {styleProps} = useStyleProps(props);
+  let formatMessage = useMessageFormatter(intlMessages);
+
+  // Sync loading state into the layout.
+  layout.isLoading = props.isLoading;
```

Comment:
if `layout` has the loading state, why are we also passing it in as a prop to CollectionView?

Reply:
CollectionView needs to know if it's already loading so it doesn't trigger duplicate `onLoadMore`